dev-server/static/scripts/pdfjs-init.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+// Note: This file is not transpiled.
+// Listen for `webviewerloaded` event to configure the viewer after its files
+// have been loaded but before it is initialized.
+document.addEventListener('webviewerloaded', () => {
+  const appOptions = window.PDFViewerApplicationOptions;
+  const app = window.PDFViewerApplication;
+
+  // Ensure that PDF.js viewer events such as "documentloaded" are dispatched
+  // to the DOM. The client relies on this.
+  appOptions.set('eventBusDispatchToDOM', true);
```

Comment:
This option has no effect any more AFAIK, it should be possible to just remove it.

Reply:
Will that impact our ability to potentially test older versions of PDF JS here?

Reply:
I don't think so. The "event bus" that this option refers to has existed for a long time.